src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -14320,31 +14335,15 @@ Map {
       "downshiftProps": Object {
         "args": Array [
           Object {
-            "breakingChanges": Object {
-              "args": Array [
-                Object {
-                  "resetInputOnSelection": Object {
-                    "type": "bool",
-                  },
-                },
-              ],
-              "type": "shape",
-            },
             "children": Object {
               "type": "func",
             },
             "defaultHighlightedIndex": Object {
               "type": "number",
             },
-            "defaultInputValue": Object {
-              "type": "string",
-            },
             "defaultIsOpen": Object {
               "type": "bool",
             },
-            "defaultSelectedItem": Object {
-              "type": "any",
-            },
```

Comment:
Change comes from Carbon, they've mitigated via a function that maps the old prop names to the new ones.